finalPrep/README.md

```diff
@@ -32,23 +32,33 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+<img width="1360" alt="Screen Shot 2021-12-18 at 2 27 36 PM" src="https://user-images.githubusercontent.com/95309774/146656221-5d6d4161-d8dd-4f01-9641-e5558b8ba070.png">
+
 - `Add Week 2 Screenshot Here`
+
+<img width="1354" alt="Screen Shot 2021-12-18 at 2 27 45 PM" src="https://user-images.githubusercontent.com/95309774/146656222-24e38b61-406b-4c06-b483-8cb6da09fd81.png">
 - `Add Week 3 Screenshot Here`
 
+
+<img width="1355" alt="Screen Shot 2021-12-18 at 2 27 53 PM" src="https://user-images.githubusercontent.com/95309774/146656223-0ee3e4a0-883b-44e4-8848-b77a8c1828e6.png">
+
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here:
+  https://gist.github.com/nbalmoori/fb966cd25557ef6596ef1ef40235bf62
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
   - [ ] Add a link to your gist here:
+  https://gist.github.com/nbalmoori/893d46ff0d61404e9dfb1446ef4d15b0
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist here:
+  https://gist.github.com/nbalmoori/9cef6483b45e4b1f7e225ddb996310a9
```

Comment:
These answers make me believe you'll be very effective on teams at Turing + beyond - you clearly know your working style and needs!